app/pages/ProjectsPage.tsx

```diff
@@ -121,6 +123,12 @@ export function ProjectsPage() {
     <>
       <PageHeader>
         <PageTitle icon={<Folder24Icon />}>Projects</PageTitle>
+        <DocsPopover
+          heading="Projects"
+          icon={<Folder16Icon />}
+          summary="Projects are logical containers for managing compute, storage and network resources within a silo."
```

Comment:
Would prefer to avoid "logical", missing Oxford comma. Not sure whether should mention the silo because for end users the silo is their whole universe. On the other hand, maybe conveying that projects are children of silos is useful to do.

> Projects are containers for managing compute, storage, and network resources.

Lower priority, I think we could try being more concrete about what is in a project.

> Projects are containers for managing resources like instances, disks, and VPCs.